enterprise/server/test/integration/ci_runner/ci_runner_test.go

```diff
@@ -324,6 +324,54 @@ func singleInvocation(t *testing.T, app *app.App, res *result) *inpb.Invocation
 	return invResp.Invocation[0]
 }
 
+func TestCIRunner_RunsBashCommands(t *testing.T) {
+	wsPath := testfs.MakeTempDir(t)
+	bazelPath, err := runfiles.Rlocation(testbazel.BazelBinaryPath)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	workspaceContentsWithBashCommands := map[string]string{
+		"WORKSPACE": `workspace(name = "test")`,
+		"buildbuddy.yaml": `
+actions:
+  - name: "Show bazel version"
+    triggers:
+      push: { branches: [ master ] }
+      pull_request: { branches: [ master ] }
+    steps:
+      - run: |
+          for i in {1..2}; do
+            echo "Loop $i: ";
+            ` + bazelPath + ` version;
```

Comment:
call `bazel` here to assert that it's available in PATH?

```suggestion
            bazel version;
```